myapp/db/migrate/20221212045433_create_tasks.rb

```diff
@@ -0,0 +1,13 @@
+class CreateTasks < ActiveRecord::Migration[6.0]
+  def change
+    create_table :tasks do |t|
+      t.string :title
+      t.string :description
+      t.integer :priority
+      t.integer :status
```

Comment:
for priority and status, you probably will have few fixed values and for that maybe `tinyint` is enough.

we can do something like `t.integer :priority, limit: 1`